src/Plugins/JJMasterData.MongoDB/Repository/MongoDBDictionaryRepository.cs

```diff
@@ -10,11 +10,11 @@
 
 namespace JJMasterData.MongoDB.Repository;
 
-public class MongoDBDictionaryRepository : IDictionaryRepository
+public class MongoDbDataDictionaryRepository : IDataDictionaryRepository
 {
     private readonly IMongoCollection<MongoDBMetadata> _metadataCollection;
 
-    public MongoDBDictionaryRepository(IOptions<JJMasterDataMongoDBOptions> options)
+    public MongoDbDataDictionaryRepository(IOptions<JJMasterDataMongoDBOptions> options)
```

Comment:
The official driver call it use `MongoDB` instead of `MongoDb`, like we call `JJ` instead of `Jj`